### ISSUE TITLE
[HTML2] Not making use of method summaries

### ISSUE THREAD

**Issue by Herst** [swagger-api/swagger-codegen#3651]
##### Description

Generator "HTML2" is doing nothing using method summaries (unlike generator "HTML" or `bootprint-openapi`).

##### Swagger-codegen version

2.2.1

##### Swagger declaration file content or url

```yaml
swagger: '2.0'
info:
  version: 0.0.0
  title: Simple API
paths:
  /:
    get:
      summary: FOOBAR
      responses:
        200:
          description: OK
```
"FOOBAR" is nowhere to be found in the resulting document.

##### Suggest a Fix

Display it above the description.

**Comment by wing328** [comment]
@Herst thanks for the suggestion. May I know if you've time to contribute the enhancement?

cc @jhitchcock

**Comment by Herst** [comment]
I have limited time but it might be certainly worth to me to contribute, but I'm not sure yet whether the `HTML` generator or `HTML2` generator or `bootprint-openapi` fit my needs best yet, so first I'm going to finalize my definition and collect/report bugs on the way.

**Comment by jhitchcock** [comment]
Hey Guys, I'll pick this up ASAP :)

**Comment by wing328** [comment]
Closed via #4014
